CORE/elements/text_field.rb

```diff
@@ -18,25 +16,11 @@ def value
         watir_element.value
     end
 
-    def fill(data)
-        assert_active
-        @world.logger.action "Filling [#{@name}] with [#{data}]"
-        manually_clear if @world.configuration["BROWSER"] == "internet_explorer"
-        watir_element.set(data)
-
-        begin
-            Watir::Wait.until(1){watir_element.value == data}
-        rescue
-            raise "ERROR: Problem filling element [#{@name}] with [#{data}] value after fill was found as [#{watir_element.value}]"
-        end
-        @world.ledger.record_fill(@name, data)
-    end
-
     def manually_clear
       watir_element.click
       browser.send_keys(:end)
       watir_element.value.size.times do
-          browser.send_keys(:backspace) unless watir_element.value == ""
+          browser.send_keys(:backspace) unless watir_element.value.chomp == ""
```

Comment:
Why did we add this chomp here? I don't think this should be a thing. If the element has empty whitespace, we still want to clear that whitespace out, not ignore it.

Reply:
that should not be there. will take it out. we were debugging and we put it in for some reason